app/kuma-dp/cmd/run.go

```diff
@@ -54,5 +54,12 @@ func newRunCmd() *cobra.Command {
 			return nil
 		},
 	}
+
+	cmd.PersistentFlags().StringVarP(&cfg.Dataplane.Name, "name", "", "", "Name of the Dataplane")
```

Comment:
`StringVarP` means that you want a single letter shortcut for that command, but you actually pass `""`.

Use `StringVar` instead